lib/builders/gpu-builder.ts

```diff
@@ -0,0 +1,169 @@
+import * as blueprints from '../../lib';
```

Comment:
remove the global blueprints reference and use individual classes. Check `observability-builder.ts` for reference in current PR.